persistence-modules/spring-jooq/pom.xml

```diff
@@ -195,11 +191,11 @@
     </build>
 
     <properties>
-        <org.jooq.version>3.16.18</org.jooq.version>
+        <org.jooq.version>3.18.7</org.jooq.version>
         <lifecycle-mapping.version>1.0.0</lifecycle-mapping.version>
         <sql-maven-plugin.version>1.5</sql-maven-plugin.version>
         <properties-maven-plugin.version>1.0.0</properties-maven-plugin.version>
-        <h2.version>2.1.214</h2.version>
```

Comment:
delete commented h2 line

Reply:
Also remove 
<version>${h2.version}</version>-->

in line 40

Reply:
Try removing also:
spring-test
spring-jdbc
spring-context

Reply:
Done.
Cannot delete spring-test, spring-jdbc and, spring-context because they are either used in an article or in a test class.